src/tourManager.ts

```diff
@@ -84,6 +98,57 @@ export class TourManager implements ITourManager {
     return this._tours;
   }
 
+  /**
+   * Extension translation bundle
+   */
+  get translator(): TranslationBundle {
+    return this._trans;
+  }
+
+  /**
+   * Creates an interactive TourHandler object that can be customized and run by the TourManager.
+   * @param tour The tour to be created.
+   *
+   * @returns The tour created
+   *
+   * #### Notes
+   * Tour title and steps content will go through a translation bundle if they are strings.
+   */
+  addTour(tour: ITour): ITourHandler | null {
+    try {
+      let trans = this._trans;
+      if (tour.translation) {
+        trans = this._translator.load(tour.translation);
+      }
+
+      const handler = this.createTour(
+        `${USER_PLUGIN_ID}:${tour.id}`,
```

Comment:
Oops, looks like this maybe shouldn't have snuck in here?

Reply:
Oh yes, thanks for catching it. I'll push a fix